utils/jwe.ts

```diff
@@ -0,0 +1,59 @@
+import * as E from "fp-ts/Either";
+import * as TE from "fp-ts/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import * as jose from "jose";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { Second } from "@pagopa/ts-commons/lib/units";
+
+/**
+ * JWE Generation
+ */
+export type GetGenerateJWE = <T extends Record<string, unknown>>(
+  issuer: NonEmptyString,
+  jweKey: NonEmptyString
+) => (payload: T, ttl: Second) => TE.TaskEither<Error, NonEmptyString>;
+
+const secondsFromEpoch = (secondsToAdd: number): Second => {
+  const date = new Date();
+  const today = Math.floor(date.getTime() / 1000);
+  const total = today + secondsToAdd;
+  return total as Second;
+};
```

Comment:
We should use the lib [date-fns](https://date-fns.org/) instead implement custom functions to works with dates.